frontend/packages/dev-console/src/components/monitoring/alerts/MonitoringAlerts.tsx

```diff
@@ -138,10 +98,10 @@ export const MonitoringAlerts: React.FC<props> = ({ match, rules, filters, listS
     dispatch(
       sortList(
         reduxID,
-        monitoringAlertColumn[index - 1].fieldName,
-        monitoringAlertColumn[index - 1].sortFunc,
+        MonitoringAlertColumn(t)[index - 1].fieldName,
+        MonitoringAlertColumn(t)[index - 1].sortFunc,
         direction,
-        monitoringAlertColumn[index - 1].title,
+        MonitoringAlertColumn(t)[index - 1].title,
```

Comment:
@vikram-raj But can you please call this method once or memorize this output of `MonitoringAlertColumn(t)`. Also if this method just return a static array we should not create this multiple times per rendering.

```tsx
const monitoringAlertColumn = MonitoringAlertColumn(t);
```

or

```tsx
const monitoringAlertColumn = React.useMemo(() => MonitoringAlertColumn(t), [t]);
```

Memo is option from my perspective here.

Reply:
Updated it. Thanks